Remove use of feature flag in credentials that we don't use any more.

matthewkmayer opened the pull request:
This should fix our broken nightly builds.

matthewkmayer commented:
This change wasn't sufficient. 

This issue should get it sorted: https://github.com/huonw/travis-cargo/issues/33 .

indiv0 commented:
Personally I would re-instate a blank `unstable` feature instead, because that attribute is intended to ensure that we don't allow warnings on stable and beta, and I think it'd be best to keep it.

Adding an empty `unstable = []` to the `Cargo.toml` should fix the problem.

matthewkmayer commented:
Ah, my understanding of it wasn't nuanced enough.  I thought if we removed the line it'd enforce `deny_warnings` by default since that's probably what we want.  

Added the feature back into the Cargo file.

matthewkmayer commented:
Nightly is "fixed."  It's now failing on the well-know "out of memory" issue: https://github.com/rusoto/rusoto/issues/513 .
